backend/services/mgmt/v1alpha1/integration_tests/anonymization-service_integration_test.go

```diff
@@ -85,58 +85,6 @@ func (s *IntegrationTestSuite) Test_AnonymizeService_AnonymizeMany() {
   "active": false,
   "sports": ["basketball", "golf", "swimming"]
 }`,
-	}
-
-	resp, err := s.unauthdClients.anonymize.AnonymizeMany(
```

Comment:
just curious, but why was this call removed?

Reply:
hmm I don't see this in my changes